src/udev/udevadm-trigger.c

```diff
@@ -219,13 +227,13 @@ int trigger_main(int argc, char *argv[], void *userdata) {
                         break;
 
                 case ARG_NAME: {
-                        _cleanup_(udev_device_unrefp) struct udev_device *dev;
+                        _cleanup_(sd_device_unrefp) sd_device *dev = NULL;
 
-                        dev = find_device(optarg, "/dev/");
-                        if (!dev)
-                                return log_error_errno(errno, "unable to open the device '%s'", optarg);
+                        r = find_device(optarg, "/dev/", &dev);
+                        if (r < 0)
+                                return log_error_errno(r, "unable to open the device '%s'", optarg);
```

Comment:
not that it matters much or was related to this PR, but in the long run we should probably capitalize these strings properly (i.e. "unable → "Unable), morevoer we should probably output the error with ": %m"...